VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -139,6 +139,28 @@ public async Task<IActionResult> GetHeartbeatDataForParticipantAsync(Guid confer
             }
         }
 
-        
+        [HttpPost("{conferenceId}/participants/{participantId}/participantDisplayName")]
+        [SwaggerOperation(OperationId = "UpdateParticipantDisplayName")]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        public async Task<IActionResult> UpdateParticipantDisplayNameAsync(Guid conferenceId, Guid participantId, [FromBody] UpdateParticipantRequest participantRequest)
+        {
+            if (conferenceId == Guid.Empty || participantId == Guid.Empty)
+            {
+                return BadRequest($"Please provide a valid conference Id and participant Id");
+            }
+
+            try
+            {
+               await  _videoApiClient.UpdateParticipantDetailsAsync(conferenceId, participantId, participantRequest);
```

Comment:
Did you check if the correct display name gets passed to the live hearing after updating the display name in the video DB? Please check. Thanks.

Reply:
we pass to live hearing    ParticipantResponse wich is mapped from conference.participant and booking participant, so display name will be mapped from video database =>
         var response = new ParticipantResponse
            {
                Id = participant.Id,
                FirstName = bookingParticipant?.First_name,
                LastName = bookingParticipant?.Last_name,
                Name = participant.Name,
                Status = status,
                Role = role,
                Username = participant.Username,
                DisplayName = participant.Display_name,
                CaseTypeGroup = participant.Case_type_group,
                Representee = participant.Representee,
                ContactEmail = bookingParticipant?.Contact_email,
                ContactTelephone = bookingParticipant?.Telephone_number
            };